koku/api/report/aws/query_handler.py

```diff
@@ -668,6 +668,11 @@ def _get_sub_org_units(self, org_unit_list):
                 .distinct("org_unit_id")
             )
 
+            # This is to remove the excluded values from the sub_org_units
+            if "org_unit_id" in list(self.query_table_exclude_keys):
+                for org_id in self.parameters.get_exclude("org_unit_id"):
+                    org_unit_list.append(org_id)
```

Comment:
`QueryParams.get_exclude()` could `None` unless a default value is passed in.
```suggestion
            if "org_unit_id" in list(self.query_table_exclude_keys):
                for org_id in self.parameters.get_exclude("org_unit_id",[]):
                    org_unit_list.append(org_id)
```

This could also be written as a comprehension which would use less memory and perform better or large data sets.

```suggestion
            if "org_unit_id" in list(self.query_table_exclude_keys):
                org_unit_list.extend(org_id for org_id in self.parameters.get_exclude("org_unit_id", []))
```

Reply:
Thanks for the pointer @samdoran gonna add this to my python arsenal 🙇